integration-box/yahoo-dmp/scripts/get_presigned_url.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import requests
+import json
+from logging import DEBUG, INFO, StreamHandler, getLogger
+import digdag
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
+import pytd
+
+logger = getLogger(__name__)
+handler = StreamHandler()
+handler.setLevel(INFO)
+logger.setLevel(INFO)
+logger.addHandler(handler)
+logger.propagate = False
+
+def generate(
+        yahoo_api_url,
+        tag_definition_guid,
+        vendor_guid,
+        entity_id,
+        uid_key,
+        brand_guid,
+        tag_fields_p,
+        tag_fields_lid):
+
+  post_data = {
+    "tagDefinitionGuid": tag_definition_guid,
+    "vendorGuid": vendor_guid,
+    "entityId": entity_id,
+    "uidKey": uid_key,
+    "brandGuid": brand_guid,
+    "tagFields": {
+      "p": tag_fields_p,
+      "lid": tag_fields_lid
+    }
+  }
+
+  headers = {"x-api-key": os.getenv('x_api_key')}
+  response = requests.post(yahoo_api_url, data=json.dumps(post_data), headers=headers)
```

Comment:
you can use `json=post_data` instead.

https://requests.readthedocs.io/en/master/user/quickstart/#more-complicated-post-requests